modules/antismash/antismashlitedownloaddatabases/main.nf

```diff
@@ -0,0 +1,34 @@
+process ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES {
+    tag '$database_dir'
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::antismash-lite=6.0.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/antismash-lite:6.0.1--pyhdfd78af_0' :
+        'quay.io/biocontainers/antismash-lite:6.0.1--pyhdfd78af_0' }"
+
+    input:
+    val(database_dir)
+
+    output:
+    path(database_dir) , emit: database
```

Comment:
This probably woudn't work, as you would have a conflict between the input/output varaible (NXF wouldn't know which to pick up... they can't be the same file.